nixos/tests/vpp.nix

```diff
@@ -0,0 +1,132 @@
+# This test runs FD.io's VPP and tests routing between two virtual networks.
+# Topology is similar to the `nat` test, with a client on one network, a server
+# on another, and a VPP-powered router inbetween connecting the two.
+
+import ./make-test-python.nix (
+  { pkgs, lib, ... }:
+  {
+    name = "vpp";
+
+    meta.maintainers = with lib.maintainers; [ romner-set ];
```

Comment:
More of a nitpick, but I would also simplify this to
```nix
meta.maintainers = [ lib.maintainers.romner-set ];
```
The `with` isn't really necessary for a single entry.

Reply:
The module could have more maintainers in the future, it's also easier to read the name like this. The [package](https://github.com/NixOS/nixpkgs/blob/793cb18e02cb7f2a8139deb375a97b27e2c96990/pkgs/by-name/vp/vpp/package.nix#L122) also has a `with lib.maintainers;`, I'll leave it in here.